docs/source/conf.py

```diff
@@ -116,8 +116,10 @@
 .. _UCSC Sequence and Annotation Data: http://hgdownload.cse.ucsc.edu/
 .. _Tute's documentation: https://docs.google.com/document/d/1_Kryc4qAqw1NRezaqDJ1tXUSCbxEkKK4SSi_kZuyHtU/pub
 
+.. _Discover Published Data: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_public_data/index.html
```

Comment:
This should be deleted.  Where ever you currently use it, instead use `:doc:/use_cases/discover_public_data`.